{"uuid": "c71799df-3eef-450e-b7c1-2d474236b24f", "start": 1562387014399, "status": "passed", "fullName": "TestCases.test_login_pytest.TestLoginP#test_login_1_failed_by_fail_datas[data0]", "name": "test_login_1_failed_by_fail_datas[data0]", "stop": 1562387015659, "historyId": "d1021337b075fdb88fea8a1fac58ed32", "parameters": [{"name": "data", "value": "{'user': '13200000000', 'check': '此账号没有经过授权，请联系管理员!', 'password': 'python'}"}], "labels": [{"name": "tag", "value": "login"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "subSuite", "value": "TestLoginP"}, {"name": "host", "value": "G50-75M"}, {"name": "thread", "value": "17676-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login_pytest"}]}